ipinfo/cmd_tool_prefix_addr.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/ipinfo/cli/lib"
+	"github.com/ipinfo/cli/lib/complete"
+	"github.com/ipinfo/cli/lib/complete/predict"
+	"github.com/spf13/pflag"
+)
+
+var completionsToolPrefixAddr = &complete.Command{
+	Flags: map[string]complete.Predictor{
+		"-h":     predict.Nothing,
+		"--help": predict.Nothing,
+	},
+}
+
+func printHelpToolPrefixAddr() {
+	fmt.Printf(
+		`Usage: %s tool prefix addr <cidr>
+
+Description:
+	returns prefix's IP address.
```

Comment:
Please dont use real tab chars in help msgs and also same comment here about a better description for this command.